mountpoint-s3/src/bin/mock-mount-s3.rs

```diff
@@ -32,7 +34,11 @@ fn create_mock_client(args: &CliArgs) -> anyhow::Result<(ThroughputMockClient, T
 
     tracing::warn!("using mock client");
 
-    let max_throughput_gbps = args.maximum_throughput_gbps.unwrap_or(10) as f64;
+    let Some(max_throughput_gbps) = args.maximum_throughput_gbps else {
+        return Err(anyhow!(
+            "must set --max-throughput-gbps when using mock-mount-s3 binary"
```

Comment:
This doesn't match the real argument.
```suggestion
            "must set --maximum-throughput-gbps when using mock-mount-s3 binary"
```